### PR TITLE
[Rollout] Production rollout

### PR DESCRIPTION
**Pull request by riarenas**
Prepping the PR for rollout this week.

### CODE REVIEW

**Comment by riarenas** [comment]
Planning to merge this right after our standup and get this rolling!

**Comment by riarenas** [comment]
@epananth I also did a local run or the scenario tests against Prod and they all passed, so we can be sure that we didn't leave anything behind from the runs you were trying to debug.

**Comment by epananth** [comment]
Thanks @riarenas :)

**Comment by riarenas** [comment]
Will wait for checks to finish and then merge.

**Comment by riarenas** [comment]
@ChadNedzlek https://dev.azure.com/dnceng/internal/_build/results?buildId=477962&view=results failed due to:

```The pipeline is not valid. Stage predeploy: Variable group DotnetStatus Production could not be found```

**Comment by riarenas** [comment]
I'm trying to create the variable group but trying to authorize against the keyvault seems to be stuck

image

**Comment by ChadNedzlek** [comment]
I made the group, so that part should work.

**Comment by riarenas** [comment]
Kicking it off again

**Comment by riarenas** [comment]
New error:

```

There was a resource authorization issue: "The pipeline is not valid. Job updateDatabase: Step AzurePowerShell1 input ConnectedServiceNameARM references service connection NetHelix which could not be found. The service connection does not exist or has not been authorized for use. For authorization details, refer to https://aka.ms/yamlauthz.,Job updateDatabase: Step AzurePowerShell2 input ConnectedServiceNameARM references service connection NetHelix which could not be found. The service connection does not exist or has not been authorized for use. For authorization details, refer to https://aka.ms/yamlauthz."
```

I clicked the authorize button and got a "Resources are now authorized" message a few times and tried requeueing but always got the same problem. At first I thought it was just trying to authorize different resources each time, but it's always the same. I ended up queueing a bunch of failed builds :| Looking.

**Comment by riarenas** [comment]
Ok, at least the build is now finally running: https://dev.azure.com/dnceng/internal/_build/results?buildId=478067&view=results

**Comment by riarenas** [comment]
Looks like @MattGal ran a second build: https://dev.azure.com/dnceng/internal/_build/results?buildId=478077&view=results.

I'll cancel mine so that we only deploy the latest.

**Comment by riarenas** [comment]
Failure deploying the dotnet-status web app: 

```
##[error]Error: Failed to fetch App Service 'dotneteng-status-staging' publishing credentials. Error: The Resource 'Microsoft.Web/sites/dotneteng-status/slots/staging' under resource group 'monitoring' was not found. (CODE: 404)
```

File: eng/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      artifact: DotNetStatus
+    - task: AzureRmWebAppDeployment@4
+      inputs:
+        ConnectionType: AzureRM
+        azureSubscription: ${{ parameters.Subscription }}
+        appType: webApp
+        WebAppName: ${{ parameters.DotNetStatusAppName }}
+        deployToSlotOrASE: true
+        ResourceGroupName: monitoring
+        SlotName: staging
```
Comment (adiaaida) [inline eng/deploy.yaml]: Should the be production for Prod?

Comment (riarenas) [reply eng/deploy.yaml]: I think this is the web app service staging slot, I think staging is still right, but we didn't add a staging slot to that web service. I believe @ChadNedzlek added one and I'mretrying that stage now

Comment (ChadNedzlek) [reply eng/deploy.yaml]: This staging means what staging is really supposed to mean. It's the place you "stage" something you are rolling out. Once it's "staged", you swap it into production in a seamless way.

The "our team" use of the word staging is what most services would call "int" or "pre-production".  Which is definitely super confusing. :-)

**Comment by riarenas** [comment]
Everything but the dotnet-status web app got deployed. I'm retrying the dotnet-status site now, and that should also kick off the scenario tests. If anyone is interested to spot check their changes before the scenario tests run, they should be able to do so now.

**Comment by riarenas** [comment]
Dotnet-status now also deployed successfully, and the scenario tests are now running

**Pull request merged**